src/platform/export/fileConverter.ts

```diff
@@ -100,7 +102,10 @@ export class FileConverter implements IFileConverter {
         candidateInterpreter?: PythonEnvironment
     ) {
         // For web, we perform plain export for Python
-        if (format === ExportFormat.python) {
+        if (
+            format === ExportFormat.python &&
+            this.configuration.getSettings(sourceDocument.uri).pythonExportMethod !== 'nbconvert'
```

Comment:
This was incorrect, not sure how the tests were passing, basically we should be using `nbconvert` when the setting is set to nbconvert. we weren't doing that.
& we also have tests for that, which was broken.